### ISSUE TITLE
New Alex graphic

### ISSUE THREAD

**Issue by Quintus** [Secretchronicles/TSC#647]
The current protagonist's graphic still resembles Mario too much and it is (sans the M on the head) the SMC protagonist graphic. I think that an entirely new graphic would benefit the game definitely.

The new appearance should probably drop red as the main colour. Also, since the current story draft depicts Alex as a treasure hunter (and I don't expect that to change), his clothing should reflect that.

**Comment by xet7** [comment]
@Quintus 

What do you think about this for new Alex graphic? Local team here is developing new game and they have graphic artist that has drawn these. It's possible to get more graphics from him. I could ask about permission to use it in TSC.

https://github.com/umpikuja/laiva/blob/master/concept-art/pelihahmonkavely.png

**Comment by Quintus** [comment]
For later reference, the picture looks like this (hint: prefix image links with a ! in markdown, this embeds the graphic):

![Concept art](https://github.com/umpikuja/laiva/raw/master/concept-art/pelihahmonkavely.png)

I think it goes toward the right direction. I like the fierce look this person shows, and the hat is cool. There could be some features added: as a treasure hunter he probably has a rucksack, and I support finger-free gloves. I know the current Alex has finger-free gloves, but I think it's reasonable for a treasure hunter -- just don't make them white as currently. It should be considered that Alex could hold items like [a flashlight](https://github.com/Secretchronicles/TSC/issues/648). Current Alex changes appearance with powerups (hello Mario...), which we should probably get rid of with the new concept.

I think the suggested concept is to tall. It wouldn't currently fit the size of the game assets. If you take the size of the current ["Big Alex" graphics](https://github.com/Secretchronicles/TSC/tree/9d0c1be6e2a12c1ad3ad6d90e9a3e56e47f9f87a/tsc/data/pixmaps/alex/big) as a measure, you probably get a good fit. Small Alex will vanish eventually in favour of our planned energy system anyway so there's not much point in keeping it around as a measure. Better have Alex crouch for narrow passages (needs graphics of course).

One thing to consider for any character graphic: the style should fit together, i.e. Dr.M., Alice, and the other persons should not use an entirely different drawing style. As the only existing person graphic (Alex) is to be replaced, the artist is free in choosing the style mostly (as long as it fits the rest of the game), but (s)he should keep in mind that there are more persons to be required to be drawn in a similar style.

> Local team here is developing new game and they have graphic artist that has drawn these. It's possible to get more graphics from him. I could ask about permission to use it in TSC.

Oh great. If @Bugsbane is no more with us, we need to find alternatives. I dropped @Skarfester an e-mail recently, but he once pointed out he doesn't feel good at complex graphics like characters. I don't know whether that changed or not. As of now, we are without a graphics artist, and that's a pretty large problem for any further development of TSC. If there's a chance to get one involved, we should really take it.

**Comment by Quintus** [comment]
Oh, and when linking to images, please always use the actual commit URL, not just the link to "master". If the graphic changes later during the discussion process, this ticket is going to look very weird.

**Comment by datahead8888** [comment]
@xet7, do you know if the artist has read the story draft?  If not, it's probably best to see if they can at least skim it before doing much prototyping.

I had thought it looked like a young boy (not 20 years old) when I first saw it.  I'd also suggest not always have him smile, but it depends on what we're going for and how cute/serious we want to aim for.

It's good to see someone taking a look at the art again :)

**Comment by xet7** [comment]
@datahead8888 

No, the artist has not read story draft, and I have not yet asked from him about TSC. That character is related to other game being developed, in that game there is old boat. Character wakes up in lowest floor of boat and uses flashlight to find something in the the dark, and does some puzzles to get to next level. These graphics have been drawn specifically for this boat game.

Artist has made some comic books that has old cards, old houses etc in cartoon style. There is a lot of different characters in those books. He makes drafts using pen and paper and then finishes bitmap art using MS Paint.

Here is that [boat game roadmap](https://github.com/umpikuja/laiva/wiki/Roadmap).

Currently that game is developed with Godot, [here is some test levels](https://github.com/umpikuja/godot-game-example).

Each level has one background image, for example this:

![One level](https://github.com/umpikuja/laiva/raw/master/concept-art/pataruuma.png)

**Comment by xet7** [comment]
Oops, typo. I above I did try to write that artist has made comic book where is old cars and houses. Not cards. Sorry, I have written too many GitHub issues about kanban cards, so it's hard to write car.

**Comment by Quintus** [comment]
> Each level has one background image, for example this:

This is interesting, as the concept is similar to how I imagine the story scenes (see #111). For the actual styling, we'll have to see.

One note: obviously we can't just re-use other games' characters. If the concept art above gets used in the boat game, we can't use it in TSC. But still it shows a possible direction.

**Comment by xet7** [comment]
There is also 3rd game (in addition to TSC+boat game) where idea is to have some backgrould image and flashlight to find something. That 3rd game is at very early planning stage currently, nothing is decided yet.

**Comment by Quintus** [comment]
No artist available; postponing.
